libcodechecker/orm_model.py

```diff
@@ -117,62 +117,60 @@ class BugPathEvent(Base):
     __tablename__ = 'bug_path_events'
 
     id = Column(Integer, autoincrement=True, primary_key=True)
+
     line_begin = Column(Integer)
     col_begin = Column(Integer)
     line_end = Column(Integer)
     col_end = Column(Integer)
+
+    order = Column(Integer)
+
     msg = Column(String)
     file_id = Column(Integer, ForeignKey('files.id', deferrable=True,
                                          initially="DEFERRED",
                                          ondelete='CASCADE'), index=True)
+    report_id = Column(Integer, ForeignKey('reports.id', deferrable=True,
```

Comment:
Do we need an id for this table? Maybe report_id, order pair could be together a key.